src/modules/search/components/SearchResultsHeader.tsx

```diff
@@ -65,6 +73,20 @@ const SearchResultsHeader = ({
             </ToggleButton>
           </ToggleButtonGroup>
         )}
+        <TextField
+          value={sortOrder}
+          onChange={(e) =>
+            onChangeSortOrder(e.target.value as ClientSortOption)
+          }
+          select
+          label='Sort Order'
+        >
+          {Object.values(ClientSortOption).map((sortOrder) => (
+            <MenuItem key={sortOrder} value={sortOrder}>
+              {HmisEnums.ClientSortOption[sortOrder]}
+            </MenuItem>
+          ))}
+        </TextField>
```

Comment:
Let's use the `GenericSelect` input so that the styling is consistent with other dropdowns.


Here's a working example. The generated `HmisEnums` object isn't typed correcty yet (but it could be!) so some casting is needed for the option list:



```
        <GenericSelect<ClientSortOption, false, false>
          sx={{ width: 250 }}
          getOptionLabel={(option) => HmisEnums.ClientSortOption[option]}
          label='Sort Order'
          onChange={(_e, value) => value && onChangeSortOrder(value)}
          value={sortOrder}
          options={
            Object.keys(HmisEnums.ClientSortOption) as ClientSortOption[]
          }
        />
```

This will lift the label up above, so we can you add a label for the display option as well so they are aligned? Like this:

![Screen Shot 2022-12-16 at 10 46 38 AM](https://user-images.githubusercontent.com/5333982/208138867-68f9210c-d585-456f-a407-543c0d6784c4.png)

Reply:
Fixed